### DISCUSSION TITLE
Go to line of search result hit in find results buffer

### DISCUSSION THREAD

**Discussion by ottob** [zed-industries/zed#11906] [Feature Requests]
First of all, thanks for a really nice editor.

Maybe im missing something obvious here. But ive done a `Find in project` search and get some results in a buffer like in the screenshot below:

If I click the filename in the header it opens the file buildSaveProfileOnInquisitionExport but it opens on line 1.
How can I open the file and directly go to line 85?
<img width="814" alt="Screenshot 2024-05-16 at 14 02 03" src="https://github.com/zed-industries/zed/assets/67104/b227eec5-7525-4682-8268-acc3e529f665">

**Comment by SomeoneToIgnore** [comment]
Question: if that thin line between excerpts would be thicker (maybe, included some line numbers in the middle?), would it be noticeable that it could be clicked?

We work on the discoverability of the feature, but so far it seems to have issues, alas.
Still, even now you can either click that small separator line to jump to the match, or use `alt-enter` in any place of this multibuffer to jump into the line the caret is on.

**Reply by ottob** [reply]
Thanks for your reply.

Ah, I see. Yeah, that line wasn't obvious :)

What confuses me about that isn't just that its almost invisible, but that I have two hits in the file, but there is only one line like that. Isn't that unexpected? 

Its also a bit too far from the hit to give good affordance I think.

My first guess was to look by the line numbers on the left to find some kind of arrow/navigate functionality there.

**Reply by kantuni** [reply]
> My first guess was to look by the line numbers on the left to find some kind of arrow/navigate functionality there.

same here

**Reply by ottob** [reply]
maybe something like this?

![Screenshot 2024-11-13 at 13 32 19](https://github.com/user-attachments/assets/4b9a678e-2cb3-4f3f-8442-dd64d6c55fd4)

**Comment by sazarkin** [comment]
Had to google this answer to understand how search works

**Comment by qwexs** [comment]
Set up a hotkey: { "f4": "editor::OpenExcerpts" }

**Reply by miles-byted** [reply]
This is the exactly correct answer.
